src/main/java/be/seeseemelk/mockbukkit/entity/OfflinePlayerMock.java

```diff
@@ -73,15 +74,13 @@ public boolean isOnline()
 	@Override
 	public boolean isOp()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.operator;
 	}
 
 	@Override
 	public void setOp(boolean value)
```

Comment:
I haven't looked at the Spigot impl but shouldn't this also set the online player's op status? We'd need to keep some sort of op list in the server that's UUID based.

Reply:
Not that I'm aware of, but I could be wrong

Reply:
In the NMS PlayerList, there's is an `ops` field which stores all opped players by their GameProfile, mapping it to their op level and whether they can bypass the player limit. We can just add a set of UUIDs to our PlayerListMock since we don't have the other concepts implemented.

Reply:
I'll do that